recipes/metachip/pre-unlink.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+rm $PREFIX/bin/MetaCHIP
```

Comment:
What's this needed for?

Reply:
Nothing. I created this file because I thought the script should be manually copied. I forgot to delete this one.